### PR TITLE
Ordering days on parse()

### PR DESCRIPTION
**Pull request by yoav-melamed**
Add relevant test under the 'At' section

### CODE REVIEW

File: src/cronParser.ts [hunk 1]
```diff
@@ -40,6 +40,25 @@ export class CronParser {
...
+        const arrayElement = parsed[i]
+          .split(",")
+          .map((item) => item.trim())
+          .filter((item) => item !== "")
+          .map((item) => (!isNaN(Number(item)) ? Number(item) : item))
+          .filter((item) => item !== null && item !== "");
+
+        if (arrayElement.length === 0) {
+          arrayElement.push("*");
+        }
```
Comment (bradymholt) [inline src/cronParser.ts:55]: What is this for?

Comment (yoav-melamed) [reply src/cronParser.ts:55]: To support the following:
`it("dayOfWeek specified as comma", function () {
      assert.equal(new CronParser("*/5 * * * * ,").parse()[5], "*");
    });`

Comment (bradymholt) [reply src/cronParser.ts:55]: Is that a valid syntax for a cron?

Comment (yoav-melamed) [reply src/cronParser.ts:55]: `*/5 * * * * ,` is an invalid syntax, this is why it parsed to `*/5 * * * * *` which is a valid syntax.
anyhow, it was an existing test that I didn't want to break.
My challenge was to add this fix while keeping all the 212 (+1 the one I added) running.

**Comment by bradymholt** [comment]
Thanks!

**Pull request merged**